### ISSUE TITLE
Problem with 240hz Monitor

### ISSUE THREAD

**Issue by voltexde** [AlmasB/FXGL#884]
https://www.youtube.com/watch?v=lZFRRC9sfac

**Comment by AlmasB** [comment]
@voltexde once the CI build finished, please can you check the issue is fixed.

To get the latest build, you can use this:
```
<dependencies>
    <dependency>
        <groupId>com.github.almasb</groupId>
        <artifactId>fxgl</artifactId>
        <version>dev-SNAPSHOT</version>
    </dependency>
</dependencies>

<repositories>
    <repository>
        <id>oss.sonatype.org-snapshot</id>
        <url>http://oss.sonatype.org/content/repositories/snapshots</url>
    </repository>
</repositories>
```

**Comment by meronbrouwer** [comment]
@AlmasB I have this same issue in a project I'm working on (https://github.com/han-yaeger/yaeger), but I don't have the hardware to test your fix. While looking into it, I get the impression that this behaviour should already be in the JavaFX AnimationTimer, and the bug should be fixed at that level. What are your thoughts on this?

**Comment by AlmasB** [comment]
[This test](https://twitter.com/Raumzeitfalle/status/1464324311687344128) confirms that the fix works as expected, at least in that one instance -- more tests from the community are appreciated. However, based on example, the issue is closed.

@meronbrouwer from the JavaFX perspective, it is not a bug. By default, AnimationTimer attempts to match the refresh rate. Any code using AnimationTimer and expecting 60fps will not work correctly. So the bug was inside the engine, not JavaFX.

The fix above removes the `0.016(6) / 60fps` expectation, by counting how many frames have passed in 1 second and then calculating `tpf = time per frame`, which is `1 / fps`. Hope this will help with your project too.

**Comment by meronbrouwer** [comment]
@AlmasB I was under the impression that the AnimationTimer interval is programmatically capped at 60fps (this stackoverflow post also points in that direction: https://stackoverflow.com/questions/45812036/javafx-60-fps-frame-rate-cap-doesnt-work/45827990#45827990), but apparently I'm mistaken.

Up until now I have been looking at JavaFX for a fix, but since your solution is also applicable to my situation, so I'll take the same approach. 
Thanks!
